src/InterventionBackend.php

```diff
@@ -76,124 +71,78 @@ class InterventionBackend implements Image_Backend, Flushable
     /**
      * Configure where cached intervention files will be stored
      *
-     * @config
-     * @var string
      */
-    private static $local_temp_path = TEMP_PATH;
+    private static string $local_temp_path = TEMP_PATH;
 
-    /**
-     * @var AssetContainer
-     */
-    private $container;
+    private ?AssetContainer $container = null;
```

Comment:
This (and some of the other properties below) need to have an explicit default value set to avoid the following error:
> Typed property must not be accessed before initialization